src/transformers/modeling_tf_t5.py

```diff
@@ -1268,13 +1267,6 @@ def call(
         else:
             input_ids = inputs
 
-            if "past_key_value_states" in kwargs:
-                warnings.warn(
-                    "The `past_key_value_states` argument is deprecated and will be removed in a future version, use `past_key_values` instead.",
-                    FutureWarning,
-                )
-                past_key_values = kwargs.pop("past_key_value_states")
-
```

Comment:
No need to remove this until we change major version.

Reply:
The `**kwargs` parameter has to be removed anyway (it brings few issues) so it should be a good move to remove it for the major version.